bin/support/bash_functions.sh

```diff
@@ -30,7 +30,14 @@ install_bootstrap_ruby()
 {
   local bin_dir=$1
   local buildpack_dir=$2
-  local heroku_buildpack_ruby_dir="$buildpack_dir/vendor/ruby/$STACK"
+
+  # Multi-arch aware stack support
+  if [ "$STACK" == "heroku-24" ]; then
+    local arch=dpkg --print-architecture
```

Comment:
This needs wrapping in `$()` otherwise `arch` will be a string containing the command name/args, rather than the output of executing the command.

```suggestion
    local arch=$(dpkg --print-architecture)
```